scripts/noaa/gpcc_spi/manifest.json

```diff
@@ -0,0 +1,112 @@
+            "scripts": [
+                "download.py",
+                "preprocess_gpcc_spi.py"
+            ],
+            "source_files": [
+                "input_files/*"
+            ],
+            "import_inputs": [
+                {
+                    "template_mcf": "gpcc_spi.tmcf",
+                    "cleaned_csv": "output_files/*.csv"
+                }
+            ],
+            "cron_schedule": "0 9 * * 2"
+        },
+        {
+            "import_name": "NOAA_GPCC_StandardardizedPrecipitationIndex_AggPlace",
+            "curator_emails": [
+                "support@datacommons.org"
+            ],
+            "provenance_url": "https://www.drought.gov/data-maps-tools/global-precipitation-climatology-centre-gpcc-standardized-precipitation-index-spi",
+            "provenance_description": "SPI for 1 degree grids calculated from monthly precipitation data using the Pearson distribution.",
+            "scripts": [
+                "download.py",
+                "preprocess_gpcc_spi.py",
+                "gpcc_spi_aggregation.py"
+            ],
+            "source_files": [
+                "input_files/*"
+            ],
+            "import_inputs": [
+                {
+                    "template_mcf": "gpcc_spi_aggregation.tmcf",
+                    "cleaned_csv": "output_files/aggregations/*.csv"
+                }
+            ],
+            "cron_schedule": "0 10 * * 2"
+        },
+        {
+            "import_name": "NOAA_GPCC_StandardardizedPrecipitationIndex_Event_Places",
+            "curator_emails": [
+                "support@datacommons.org"
+            ],
+            "provenance_url": "https://www.drought.gov/data-maps-tools/global-precipitation-climatology-centre-gpcc-standardized-precipitation-index-spi",
+            "provenance_description": "SPI for 1 degree grids calculated from monthly precipitation data using the Pearson distribution.",
+            "scripts": [
+                "run.sh"
+            ],
+            "source_files": [
+                "input_files/*"
+            ],
+            "import_inputs": [
+                {
+                    "template_mcf": "drought_spi_9m_polygon_place_svobs.tmcf",
+                    "cleaned_csv": "output_files/events_spi_9m_polygon/place_svobs/drought_spi_9m_polygon_place_svobs.csv"
+                }
+            ],
+            "cron_schedule": "0 11 * * 2"
+        },
+        {
+            "import_name": "NOAA_GPCC_StandardardizedPrecipitationIndex_Event",
+            "curator_emails": [
+                "support@datacommons.org"
+            ],
+            "provenance_url": "https://www.drought.gov/data-maps-tools/global-precipitation-climatology-centre-gpcc-standardized-precipitation-index-spi",
+            "provenance_description": "SPI for 1 degree grids calculated from monthly precipitation data using the Pearson distribution.",
+            "scripts": [
```

Comment:
the run.sh can be moved to the  import above if it need that as input.
Also since run.sh generates outputs this and imports below, we can have a single import in this manifest and refer to the same latest in the manifest.textproto for other imports.